pkg/cli/command/uninstall.go

```diff
@@ -29,7 +30,7 @@ func UninstallCommand() *cobra.Command {
 			}
 		},
 	}
-	cmd.Flags().StringVar(&request.Namespace, "namespace", "default", "Namespace for uninstall")
+	cmd.Flags().StringVar(&request.Namespace, "namespace", client.GetDefaultNamespace(), "Namespace for uninstall")
```

Comment:
Could you help to also add a shortcut '-n' to the namespace flag when users do uninstall here?

Reply:
Updated

Reply:
thx!